### PR TITLE
Example scene polish cleanup

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
Updated HandInteractionExamples scene.
- Fixed Round Button movement for the far interaction
- Fixed Piano key movement for the far interaction
- Updated outdated text labels
- Added WorldAnchor to the SceneContent object for stability (@Troy-Ferrell )
- Added audio feedback to the Unity UI examples
- Renamed HolographicButton.prefab as RecordButton.prefab. It is ET demo specific button. HolographicButton was HoloLens 1 button from HTK. 

## Changes
- Fixes: #5164 and #4950 and #4280

2019-07-17 22_00_35-Unity 2018 4 2f1 Personal - HandInteractionExamples unity - MRTK-GitHub - Univer

### CODE REVIEW

**Comment by Troy-Ferrell** [comment]
Added script to handle coffee cup falling forever :)

Really great to see all these changes! Just again FYI to PR #5333

File: Assets/MixedRealityToolkit.SDK/StandardAssets/Materials/MRTK_Standard_GlowingCyan.mat [hunk 1]
```diff
@@ -9,8 +9,8 @@ Material:
   m_PrefabAsset: {fileID: 0}
   m_Name: MRTK_Standard_GlowingCyan
   m_Shader: {fileID: 4800000, guid: 5bdea20278144b11916d77503ba1467a, type: 3}
-  m_ShaderKeywords: _BORDER_LIGHT_USES_HOVER_COLOR _DISABLE_ALBEDO_MAP _HOVER_LIGHT
-    _INNER_GLOW
+  m_ShaderKeywords: _BORDER_LIGHT _BORDER_LIGHT_USES_HOVER_COLOR _DISABLE_ALBEDO_MAP
```
Comment (Troy-Ferrell) [inline Assets/MixedRealityToolkit.SDK/StandardAssets/Materials/MRTK_Standard_GlowingCyan.mat]: Is this material change unique to HandInteractionExamples or good for all uses of this standard asset?

Comment (cre8ivepark) [reply Assets/MixedRealityToolkit.SDK/StandardAssets/Materials/MRTK_Standard_GlowingCyan.mat]: This was used for the large gem in the Touch interaction example. The change is good for other examples if it is used.

**Pull request merged**